.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+**CHANGE DESCRIPTION**
+---
+**Problem:**
+*Short explanation of the problem.*
+
+**Cause:**
+*Short explanation of the root cause of the issue if applicable.*
+
+**Solution:**
+*Short explanation of the solution we are providing with this PR.*
+
+**CHECKLIST**
+---
+**Jira**
+- [ ] Is Jira ticket created and referenced properly?
+- [ ] Does Jira ticket have proper status for Documentation if needed?
+- [ ] Does Jira ticket link to proper `Fix Version` milestone?
```

Comment:
documentation or qa needed

Reply:
QA currently is getting assigned by default, but I will add it just in case if somebody accidentally removes the flag.